pkg/arvo/sys/hoon.hoon

```diff
@@ -558,6 +558,8 @@
 ::
 ++  gulf                                                ::  range inclusive
   |=  {a/@ b/@}
+  ?>  (gth b a)
```

Comment:
I'd allow the case where `=(a b)`, and probably express the opposite condition as well:

```
?>  (lte a b)
```
```